pubspec.yaml

```
@@ -20,7 +20,7 @@ dependencies:
   firebase_core: ^2.25.4
   firebase_messaging: ^14.7.15
   flutter_local_notifications: ^16.3.2
-  http: ^0.13.6
+  http: '>=0.13.6'
```

```suggestion
  http: '>=0.13.6 < 2.0.0'
```

I propose we change it to this instead just to prevent things from breaking should the next major version change be one that affects how we use the library. I know the chances of that happening are slim considering how we use the library now but I'd avoid it.

If the next major version change doesn't affect us, the user could always use `dependency_overrides` to force our SDK to use that other version with no issue.

---

OK, we can always re-evaluate this choice if/when http releases version 2